server/platform-support/platform-support.rst

```diff
@@ -13,29 +13,29 @@
    * - Platform
      - Architecture
      - Edition
```

Comment:
(question)
Is there any need for the 4.2 column any more, since it's EOL'd?

Reply:
I was originally going to remove it, but I felt it'd be best to keep five version columns for the sake of the table formatting / even spacing.